citeproc-java/build.gradle

```diff
@@ -44,7 +44,7 @@ def getV8Dependency() {
         logger.error("Could not find V8 runtime compatible to this system")
         return null
     }
-    return 'com.eclipsesource.j2v8:j2v8_' + dep + ':4.5.0'
+    return 'com.eclipsesource.j2v8:j2v8_' + dep + ':4.8.0'
```

Comment:
As I said earlier: This will fail on macOS and Windows because there is no version 4.8.0 of J2V8. The latest binaries available seem to be for version 4.6.0

Reply:
Thanks! The j2v8 4.8 version slipped through my squashing.